terminateTestProcesses.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Arg S1 : Unix current env.User
+
+# If Windows uses PowerShell
+if [ "$OSTYPE" = "cygwin" ]; then
```

Comment:
It looks like this was `PLATFORM.contains("windows")` instead of an explicitly cygwin check previously. What is the reason for this change as there are alternative environments that can be used to build on Windows other than cygwin that would presumably not be covered after this.

Reply:
I suspect that doing `if [ "$OS" = "Windows_NT" ]` would trap all situations since that's defined by default on Windows, and wouldn't require any additional parameters. `SXA-ProcessCheck` was a bit of a hack so I wouldn't necessarily assume that its way of doing things is a good example of how to do things ;-)